app/views/decidim/decidim_awesome/admin/admin_actions/index.html.erb

```diff
@@ -2,7 +2,40 @@
   <div class="card-divider">
     <h2 class="card-title"><%= t("menu.admin_accountability", scope: "decidim.admin", default: "Admin accountability") %></h2>
   </div>
+  <div class="card-section fcell grid-x">
+    <div class="cell auto text-right">
+      <%= admin_filters_pagination %>
+    </div>
+  </div>
   <div class="card-section">
-    <p>List of admin actions</p>
+    <div class="table-scroll">
+      <table class="table-list">
+        <thead>
+        <tr>
+          <th><%= t("role", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("name", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("email", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("participatory_space", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("last_login", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("created_at", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+          <th><%= t("removal_date", scope: "decidim.decidim_awesome.admin.admin_accountability") %></th>
+        </tr>
+        </thead>
+        <tbody>
+        <% admin_actions.each do |admin_action| %>
+          <tr>
+            <td><%= role_from_papertrail(admin_action) %></td>
+            <td><%= admin_action_user(admin_action).name %></td>
+            <td><%= admin_action_user(admin_action).email %></td>
+            <td><%= participatory_space_type(admin_action) %></td>
+            <td><%= admin_action_user(admin_action).last_sign_in_at ? I18n.l(admin_action_user(admin_action).last_sign_in_at, format: :short) : "" %></td>
+            <td><%= I18n.l(admin_action.changeset["created_at"].last, format: :short) %></td>
+            <td class="<%= currently_active == removal_date(admin_action) ? "text-success" : nil %>"><%= removal_date(admin_action) %></td>
```

Comment:
Instead of using the helpers, we should be using different methods of the presenter:

```ruby
admin_action.role
```